locations/exporters.py

```diff
@@ -138,7 +138,8 @@ def _get_serialized_fields(self, item, default_value=None, include_empty=None):
                 )
             except ValueError:
                 logging.warning("Couldn't convert lat (%s) and lon (%s) to string", lat, lon)
-
+        elif geometry := item.get("geometry"):
+            feature.append(("geometry", geometry))
```

Comment:
Can you change the lat/lon stuff above to set the `geometry` property instead of adding to the feature directly?

Reply:
Done.